### ISSUE TITLE
Update notification not working

### ISSUE THREAD

**Issue by wildonesmoot** [yeriomin/YalpStore#273]
### Expected behaviour
Check for updates in the background and display a notification.

### Actual behaviour
Notification appears only on Yalp Store launch contradictory to settings.

### Steps to reproduce
1. Set "Check for updates periodically" to "Hourly", "Daily" or "Weekly"
2. No update notification is displayed
3. Launch Yalp Store
4. Update notification is displayed

### Info about your device
Nexus 5, stock Android 6.0.1, no root, Yalp Store 0.24, built-in credentials

**Comment by yeriomin** [comment]
Cannot reproduce.

There seems to be a missing step between 1 and 2. Maybe you force closed Yalp Store? Did it crash? Maybe you forbade it to run at boot and rebooted your device?

**Comment by wildonesmoot** [comment]
I uninstalled Yalp Store 0.24 and tried with 0.25 (OTA).
I kept all settings at default values except "Check for updates periodically".
I still do not receive any update notification when a newer version of an installed app is present in the store.
When I launch Yalp Store, the update is displayed on top of the list, but no notification appears.
No force close. No crash. Did not disable run at boot. Might be related to #275?

**Comment by yeriomin** [comment]
Please:
1. Turn off periodic checks.
2. Force close Yalp Store.
3. Start it and turn periodic checks on.
4. Send me the logs.

> Might be related to #275?

Maybe.

**Comment by wildonesmoot** [comment]
```09-18 21:52:25.341   771  1939 I ActivityManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.github.yeriomin.yalpstore/.UpdatableAppsActivity (has extras)} from uid 10022 on display 0
09-18 21:52:25.375   771   782 I ActivityManager: Start proc 30333:com.github.yeriomin.yalpstore/u0a208 for activity com.github.yeriomin.yalpstore/.UpdatableAppsActivity
09-18 21:52:25.433 30333 30333 W System  : ClassLoader referenced unknown path: /data/app/com.github.yeriomin.yalpstore-2/lib/arm
09-18 21:52:25.447 30333 30333 V com.github.yeriomin.yalpstore.UpdatableAppsActivity: Starting activity
09-18 21:52:25.477 30333 30333 V com.github.yeriomin.yalpstore.UpdatableAppsActivity: Resuming activity
09-18 21:52:25.601   771   792 I ActivityManager: Displayed com.github.yeriomin.yalpstore/.UpdatableAppsActivity: +238ms
09-18 21:52:33.124   771  1939 I ActivityManager: START u0 {cmp=com.github.yeriomin.yalpstore/.PreferenceActivity} from uid 10208 on display 0
09-18 21:52:33.167 30333 30333 V com.github.yeriomin.yalpstore.UpdatableAppsActivity: Pausing activity
09-18 21:52:33.396   771   792 I ActivityManager: Displayed com.github.yeriomin.yalpstore/.PreferenceActivity: +228ms
09-18 21:52:33.932 30333 30333 V com.github.yeriomin.yalpstore.UpdatableAppsActivity: Stopping activity
09-18 21:52:37.427 30333 30333 I com.github.yeriomin.yalpstore.UpdateChecker: Enabling periodic update checks```

**Comment by yeriomin** [comment]
Should be fixed in 0.26.
